django_saml2_auth/errors.py

```diff
@@ -0,0 +1,27 @@
+"""Error codes used for reporting errors to user."""
+
+EMPTY_FUNCTION_PATH = 5000
```

Comment:
It's not defined in SAML whatsoever, rather I thought counting from 5xxx would be nice. Also, this is a public library and it's error codes are reported with a custom HTML template (`error.html`), which I think might not clash with our own error code, since it doesn't follow or trigger `APIError`. Anyhow, if you still think that I should follow those error codes, I'd change them.

Reply:
The use of the error code has less to do with clashes and with what we (eventually) will be able to tell users about the error codes we return. While it's a little strange for a public lib like this, it wouldn't hurt to make the error codes match up with our other codes when we have the chance.

Reply:
I used 11xx instead.